androidsdk/src/main/java/com/wootric/androidsdk/Constants.java

```diff
@@ -30,4 +30,8 @@
     public static final long DAY_IN_MILLIS = 1000L *60L *60L *24L;
 
     public static final int NOT_SET = -1;
+
+    public static final long DEFAULT_RESURVEY_DAYS = 90L;
```

Comment:
Does this work similar to how web beacon works in terms of cookie expiry? So if resurvey_throttle is set to 90 days, SDK would not event hit our eligibility servers for 90 days.